lib/led-matrix/include/LEDMatrix.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include "hw_bridge.h"
+
+// Flags
+#define START_FLASH 1UL  // 01
+#define ENDED_FLASH 2UL  // 10
```

Comment:
Suggestion: make these constexpr private static variables instead

Reply:
Thanks for the suggestion, I've updated it!